scripts/build-and-pack.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Stop on error.
+set -e
+
+# Echo commands.
+set -x
```

Comment:
`set -x` is for debug purpose in my mind. I would suggest removing it in official code to prevent printing unexpected content in future.

Reply:
good point. I'll explicit echo commands instead.